en/docs/learn/design-api/publish-api/publish-an-api-with-artifact-synchronization.md

```diff
@@ -0,0 +1,158 @@
+#Publish an API with Artifact Synchronization
+
+Currently, in a Multi Gateway setup, synapse artifacts such as sequences, local entries, endpoints are saved in
+` <APIM_HOME>/repository/deployment/server/synapse-configs/default` directory as XMLs and have to be synced between all the gateway nodes using NFS or rsync. 
+When using NFS we need to manage additional components that result in a considerable amount of changes in the current architecture.
+
+Thus, a solution with an extension point which can be configurable to store these synapse artifacts is introduced.
+
+
+##API Publish/ Update / Remove when the API Gateway is running
```

Comment:
Again the title here does not give out the idea that it intends to.

Reply:
Here we discuss about the updates (Publish/update/remove) from the publisher when the gateway is up .